interfaces/builtin/common.go

```diff
@@ -162,8 +163,14 @@ func (iface *commonInterface) SecCompConnectedPlug(spec *seccomp.Specification,
 }
 
 func (iface *commonInterface) UDevConnectedPlug(spec *udev.Specification, plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot) error {
-	for _, rule := range iface.connectedPlugUDev {
-		spec.TagDevice(rule)
+	// don't tag devices if the interface controls it's own device cgroup
+	if !iface.controlsDeviceCgroup {
```

Comment:
Nitpick: could you please drop the negation and simply swap the if / else code inside.

Reply:
Done